lambda/MAS/Services/S3Service.cs

```diff
@@ -1,18 +1,16 @@
-﻿using Amazon.Lambda.Core;
-using Amazon.S3;
+﻿using Amazon.S3;
 using Amazon.S3.Model;
 using MAS.Configuration;
 using MAS.Models;
 using Microsoft.Extensions.Logging;
 using System;
-using System.Text;
 using System.Threading.Tasks;
 
 namespace MAS.Services
 {
     public interface IS3Service
```

Comment:
I still think this should be called IStaticContentService or something to be independent of the fact that we're using S3. The class itself can still be called S3Service as that's a concrete implementation